azero/contracts/gas-price-oracle/contract/lib.rs

```diff
@@ -6,13 +6,15 @@ pub mod oracle {
     use ink::{
         env::{set_code_hash, Error as InkEnvError},
         prelude::{format, string::String},
+        storage::{traits::ManualKey, Lazy},
     };
+    use ownable::*;
     use scale::{Decode, Encode};
 
     #[ink(storage)]
     pub struct Oracle {
-        /// oracle owner
-        owner: AccountId,
+        /// data for Ownable2Step - oracle owner
+        ownable_data: Lazy<ownable::Data, ManualKey<0xDEADBEEF>>,
```

Comment:
I don't think `ownable_data` needs to be `Lazy` here

Reply:
Correct me if I'm wrong: Assuming that `OwnableData` struct won't change in any potential contract update that's right. We can get rid of Lazy here.